apps/condo/domains/property/components/panels/Builder/MapJsonSchema.json

```diff
@@ -115,6 +123,120 @@
             "title": "sections",
             "type": "array"
         },
+        "parking": {
+            "items": {
+                "additionalProperties": true,
+                "properties": {
+                    "floors": {
+                        "items": {
+                            "additionalProperties": true,
+                            "properties": {
+                                "id": {
+                                    "title": "id",
+                                    "type": "string"
+                                },
+                                "index": {
+                                    "title": "index",
+                                    "type": "number"
+                                },
+                                "name": {
+                                    "title": "name",
+                                    "type": "string"
+                                },
+                                "type": {
+                                    "enum": [
+                                        "floor"
+                                    ],
+                                    "title": "type",
+                                    "type": "string"
+                                },
+                                "units": {
+                                    "items": {
+                                        "additionalProperties": true,
+                                        "properties": {
+                                            "id": {
+                                                "title": "id",
+                                                "type": "string"
+                                            },
+                                            "label": {
+                                                "title": "label",
+                                                "type": "string"
+                                            },
+                                            "name": {
+                                                "title": "name",
+                                                "type": "string"
+                                            },
+                                            "type": {
+                                                "enum": [
+                                                    "unit"
+                                                ],
+                                                "title": "type",
+                                                "type": "string"
+                                            },
+                                            "unitType": {
+                                                "enum": [
+                                                    "flat",
+                                                    "parking"
```

Comment:
How can a flat be in parking?
Does it meant to be a living flat in parking context?

Reply:
@AntonAL fixed